src/app/pages/TransactionDetailPage/index.tsx

```diff
@@ -136,6 +137,8 @@ export const TransactionDetailView: FC<{
   const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
   const formattedTimestamp = useFormattedTimestampString(transaction?.timestamp)
 
+  const ticker = transaction ? getNetworkTickers(t)[transaction.network] : ''
```

Comment:
Will backend ever add token symbol/ticker too all APIs? If yes, then I would add it to api.ts

Reply:
Good question, I have no idea. In any case, yes, we could add the ticker in api.ts

Reply:
OK, for now I'm adding the ticker to all accounts and transactions in the API layer. Opened discussion with backend developers about including the data; if they will add it, we can remove it..